src/db/users.rs

```diff
@@ -12,6 +13,59 @@ pub struct User {
     pub id: Uuid,
     pub username: String,
     pub password_hash: String,
+    pub email: Option<String>,
```

Comment:
The compiler is emitting a warning because these four fields are unused. Since we might want to use them later, we should keep the fields but silence the warning, like this:
```suggestion
    #[allow(dead_code)]
    pub email: Option<String>,
```